src/components/shapes/TodoListShape.js

```diff
@@ -0,0 +1,6 @@
+import propTypes from 'prop-types';
+import { TodoShape } from './TodoShape';
+
+export const TodoListShape = {
+  todos: propTypes.arrayOf(TodoShape),
```

Comment:
prop-Types - it's a library name and PropTypes - it's a method (starts always with a capital letter)
fix everywhere

```suggestion
  todos: PropTypes.arrayOf(TodoShape),
```